test/fixtures/simple/index.js

```diff
@@ -9,8 +9,10 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
+const { Response } = require('node-fetch');
```

Comment:
this should automatically include node-fetch to the modules..... so no need to add it extra.

this also shows the _drawback_ of the universal approach. every action needs an extra dependency to node-fetch (or to whatever provides the `Response`).